benchmarks/auto_tune.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/bash
+
+# This script aims to tune the best server parameter combinations to maximize throughput for given requirement. 
+# The current server parameter combination is  max_num_seqs and max_num_batched_tokens
+# It also supports additional requirement: e2e latency and prefix cache. 
+
+# For example: 
+# 1. Given input_len=1800, output_len=20, what's the best max_num_seqs and max_num_batched_tokens to get highest throughput?
```

Comment:
For each of the 3 scenarios, could you give add some usage explanation, for example what is a sample command therein what are the important parameters that users need to configure?